evm/src/generation/partial_trie.rs

```diff
@@ -0,0 +1,24 @@
+use ethereum_types::U256;
+
+/// A partial trie, or a sub-trie thereof.
+pub enum PartialTrie {
```

Comment:
cc @BGluth. I changed `Leaf` as per your comment but let me know if you have other feedback on this.

Reply:
Yeah this seems good!